app/components/circle/circle.tsx

```diff
@@ -0,0 +1,176 @@
+import { View } from "react-native"
+import { Text, makeStyles, useTheme } from "@rneui/themed"
+
+import { useCountUp } from "use-count-up"
+import Icon from "react-native-vector-icons/Ionicons"
+
+type CircleProps = {
+  heading: string
+  value: number
+  minValue?: number
+  maxValue?: number
+  description: string
+  subtitle: string
+  subtitleGreen?: boolean
+  extraSubtitleLine?: string
+  bubble?: boolean
+  iBtnModal?: React.ReactElement
+  iBtnModalEnable?: () => void
+  countUpDuration?: number
+}
+
+export const Circle: React.FC<CircleProps> = ({
+  heading,
+  value,
+  description,
+  subtitle,
+  subtitleGreen = false,
+  extraSubtitleLine,
+  bubble = false,
+  minValue = 1,
+  maxValue = 100,
```

Comment:
do we need default here? are those values used at any point?

Reply:
yeah, don't need them, removed